trin-types/src/cli.rs

```diff
@@ -205,14 +205,14 @@ impl Default for TrinConfig {
 
 impl TrinConfig {
     pub fn from_cli() -> Self {
-        Self::new_from(env::args_os()).expect("Could not parse trin arguments")
+        Self::new_from(env::args_os()).unwrap_or_else(|e| e.exit())
```

Comment:
Ahh, nice! I think this is the magic. We've had some difficulty with these tests over the past. The reason for the explicit `panic!`s was due to our use of `should_panic(...)` in the tests. But, if we can use `clap`s native errors, I'm all for it.